mappings/net/minecraft/client/render/VertexFormat.mapping

```diff
@@ -31,9 +33,11 @@ CLASS net/minecraft/class_293 net/minecraft/client/render/VertexFormat
 		FIELD field_27383 mode I
 		FIELD field_27384 vertexCount I
 		FIELD field_27385 size I
+		FIELD field_38878 strip Z
```

Comment:
`strip` is misleading since `TRIANGLE_FAN` sets this field to `true`. This is whether it reuses one or more vertices. For example, `TRIANGLE_STRIP` reuses the last two vertices. `TRIANGLE_FAN` reuses the first vertex and the last vertex.

Reply:
```suggestion
		FIELD field_38878 shareVertices Z
```